api/swagger/swagger.yaml

```diff
@@ -810,6 +827,11 @@ definitions:
         type: string
       name:
         type: string
+      url:
+        type: string
+        description: URL to source's website
+      terms_and_conditions_url:
```

Comment:
I know, but I added it to most of the fields we had. It seems all the good docs out there have redundant descriptions 😕 
Plus, they look nicer in the docs (depending on the template used). And as a user I guess one would feel safer to get to a docs site where even everything is filled in (we're not there yet, but on our way).